webr-vars.mk

```diff
@@ -26,17 +23,18 @@ CXX17 = em++
 CXX20 = em++
 CC = emcc
 CXX = em++
-CFLAGS = -std=gnu11 $(EM_LIBS) $(R_INCLUDES)
-CXXFLAGS = -std=gnu++11 $(EM_CXX_FIXES) $(EM_LIBS) $(R_INCLUDES)
-CXX98FLAGS = -std=gnu++98 $(EM_CXX_FIXES) $(EM_LIBS) $(R_INCLUDES)
-CXX11FLAGS = -std=gnu++11 $(EM_CXX_FIXES) $(EM_LIBS) $(R_INCLUDES)
-CXX14FLAGS = -std=gnu++14 $(EM_CXX_FIXES) $(EM_LIBS) $(R_INCLUDES)
-CXX17FLAGS = -std=gnu++17 $(EM_CXX_FIXES) $(EM_LIBS) $(R_INCLUDES)
-CXX20FLAGS = -std=gnu++20 $(EM_CXX_FIXES) $(EM_LIBS) $(R_INCLUDES)
-
-LDFLAGS = -s SIDE_MODULE=1 -s WASM_BIGINT -s ASSERTIONS=1
-FC = $(LLVM_BUILD_DIR)/../emfc
-FLIBS = -L$(WEBR_LOCAL)/../lib -lFortranRuntime
+CFLAGS = -std=gnu11 $(EM_LIBS) $(WEBR_INCLUDES)
+CXXFLAGS = -std=gnu++11 $(EM_CXX_FIXES) $(EM_LIBS) $(WEBR_INCLUDES)
+CXX98FLAGS = -std=gnu++98 $(EM_CXX_FIXES) $(EM_LIBS) $(WEBR_INCLUDES)
+CXX11FLAGS = -std=gnu++11 $(EM_CXX_FIXES) $(EM_LIBS) $(WEBR_INCLUDES)
+CXX14FLAGS = -std=gnu++14 $(EM_CXX_FIXES) $(EM_LIBS) $(WEBR_INCLUDES)
+CXX17FLAGS = -std=gnu++17 $(EM_CXX_FIXES) $(EM_LIBS) $(WEBR_INCLUDES)
+CXX20FLAGS = -std=gnu++20 $(EM_CXX_FIXES) $(EM_LIBS) $(WEBR_INCLUDES)
+
+LDFLAGS = -s SIDE_MODULE=1 -s WASM_BIGINT -s ASSERTIONS=1 $(WEBR_LDFLAGS)
+
+FC = $(WEBR_ROOT)/tools/flang/emfc
```

Comment:
Should this read `$(WEBR_ROOT)/tools/fortran.mk` incase someone is using `$(WEBR_ROOT)/tools/dragonegg/emfc`?

Reply:
oh I guess so